src/frontend/components/component-info/component-info.ts

```diff
@@ -1,40 +1,39 @@
 declare var JSONFormatter: any;
 import {Component, ElementRef, Inject, EventEmitter,
   OnChanges}
-  from '@angular/core';
+  from 'angular2/core';
 
-import {UserActions} from '../../actions/user-actions/user-actions';
-
-import Accordion from '../accordion/accordion';
 import ParseData from '../../utils/parse-data';
 import RenderState from '../render-state/render-state';
-import Dependency from '../dependency/dependency';
 
 @Component({
   selector: 'bt-component-info',
   templateUrl: '/src/frontend/components/component-info/component-info.html',
   inputs: ['node'],
-  directives: [RenderState, Accordion, Dependency]
+  outputs: ['selectDependency'],
+  directives: [RenderState]
 })
 export default class ComponentInfo {
   private node: any;
+  private selectDependency: EventEmitter<string> = new EventEmitter<string>();
   private propertyTree: string = '';
-  private _input: Array<any>;
 
   constructor(
-    @Inject(ElementRef) private elementRef: ElementRef,
-    private userActions: UserActions
+    @Inject(ElementRef) private elementRef: ElementRef
```

Comment:
@sumitarora do we need the @Inject here? I would think that injection is happening based on type here anyway.